src/components/molecules/BenefitFilter.stories.js

```diff
@@ -31,9 +28,6 @@ Eligible.args = {
   eligibleText: "Eligible benefits",
   helpText: "Potential help",
   othersText: "Others",
-  isSelectedEligible: true,
-  isSelectedHelp: false,
-  isSelectedOthers: false,
```

Comment:
So I was trying to identify this in my other comment (maybe I wasn't super clear)..

Removing this, now has 4 identical displays for the Benefit filter, rather than 4 separate ones based on which filter is selected - which is removed functionality.

Imo there should be some indication of which filter is selected, so these should get added back. 

This removes the `invert` in the `ActionButton` in place of using the `classNames`, which would have been fine if an inverted (or selected) class styling was applied when the filter is selected.

Reply:
I see what you mean, thanks for the clarification, I put back the invert prop, the buttons are now displayed in it's selected stage in the story